BUILDING.md

```diff
@@ -0,0 +1,39 @@
+# Building JanusGraph.Net
+
+JanusGraph.Net uses [Cake][cake] for build automation to get
+deterministic builds across platforms.
+
+## Requirements
+
+* [.NET Core SDK 2.1][dotnet-sdk] is needed to
+build and test the project.
+* [Docker][docker] needs to be running in order to execute the
+integration tests as they automatically start a JanusGraph Docker container.
+
+## Build Scripts
+
+The repository contains two build scripts, one for Linux (`build.sh`) and one
+for Windows (`build.ps1`). These scripts bootstrap and execute the Cake
+script (`build.cake`) which contains different tasks for building and testing.
+
+When these scripts are executed without any arguments, then the library will be
+built and tested.
+
+Other tasks can be executed by specifying them as targets:
+
+```sh
+./build.sh --target Pack
```

Comment:
Once I got this command to work (see my other comment in `build.sh`), I noticed this warning:

```
home/ubuntu/.dotnet/sdk/2.2.100/Sdks/NuGet.Build.Tasks.Pack/buildCrossTargeting/NuGet.Build.Tasks.Pack.targets(202,5):
error NU5125: The 'licenseUrl' element will be deprecated. Consider using the 'license' element instead.
[/home/ubuntu/janusgraph-dotnet/src/JanusGraph.Net/JanusGraph.Net.csproj]
```

The `Pack` task completed successfully.

Reply:
Yeah, I think they deprecated the `licenseUrl` element after I initially created this PR, but I'll use the new element instead.